src/witan/send/settings.clj

```diff
@@ -597,8 +624,8 @@
               sen-unit-name
               resourced-provision-name]
       :or    {edubaseall-send-map      {}
-              designation-f            (fn [& args] nil)
-              area-split-f             (fn [& args] nil)
+              designation-f            (constantly nil)
+              area-split-f             (constantly nil)
```

Comment:
nice. I always forget constantly and repeatedly and repeat and which one I want to use when. Thank goodness for docs and examples.